rdr/busybox/cephfs/app-busybox-2/resources/deployment/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+resources:
+- busybox-dc-1.yaml
```

Comment:
As discussed earlier, it would be better to have 4-5Pods consuming 1 single PVC for CephFs RWX volumes. WDYT?

Reply:
Looks like we are already doing that as `claimName: busybox-pvc-1` is same for all Deployment Pods. But why do we have 4 PVC's then?